File: app/src/main/java/com/perc/pavel/sportgeolocationgame/MainMenuActivity.java
package com.perc.pavel.sportgeolocationgame;

import android.content.Intent;
import android.graphics.Color;
import android.graphics.Typeface;
import android.location.Location;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.SeekBar;
import android.widget.TextView;
import android.widget.Toast;

import org.json.JSONArray;
import org.json.JSONException;
import org.json.JSONObject;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

public class MainMenuActivity extends AppCompatActivity {
    
    TextView profileName;
    Button btnPlayGame;
    Button btnGameAuthors;
    Button btnGameRules;
    Button btnSupportAuthors;
    Profile profile;
    
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main_menu);
        profile = (Profile) getIntent().getSerializableExtra("profile");


        profileName = (TextView) findViewById(R.id.profileName);
        profileName.setText(profile.getName());

        btnPlayGame = (Button) findViewById(R.id.btnPlayGame);
        btnGameRules = (Button) findViewById(R.id.btnGameRules);
        btnGameAuthors = (Button) findViewById(R.id.btnGameAuthors);
        btnSupportAuthors = (Button) findViewById(R.id.btnSupportAuthors);

        Typeface Matiz = Typeface.createFromAsset(getAssets(), "fonts/Matiz.ttf");
        profileName.setTypeface(Matiz);
        
        Typeface PhosphateSolid = Typeface.createFromAsset(getAssets(), "fonts/PhosphateSolid.ttf");
        btnPlayGame.setTypeface(PhosphateSolid);
        btnGameRules.setTypeface(PhosphateSolid);
        btnGameAuthors.setTypeface(PhosphateSolid);
        btnSupportAuthors.setTypeface(PhosphateSolid);
        
    }


    public void btnPlayGameClick(View v) {
        Intent intent = new Intent(MainMenuActivity.this, StartGameActivity.class);
        intent.putExtra("profile", (Serializable)profile);
        
        startActivity(intent);
    }
    
    public void btnGameRulesClick(View v) {
        Intent intent = new Intent(MainMenuActivity.this, RulesActivity.class);
        startActivity(intent);
    }
}

File: app/src/main/java/com/perc/pavel/sportgeolocationgame/activities/MainMenuActivity.kt
package com.perc.pavel.sportgeolocationgame.activities

import android.content.Intent
import android.graphics.Typeface
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import com.perc.pavel.sportgeolocationgame.Profile
import com.perc.pavel.sportgeolocationgame.R
import com.perc.pavel.sportgeolocationgame.RulesActivity
import kotlinx.android.synthetic.main.activity_main_menu.*

import java.io.Serializable

class MainMenuActivity : AppCompatActivity() {
    
    private lateinit var profile: Profile
    
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main_menu)
        profile = intent.getSerializableExtra("profile") as Profile
        
        profileName.text = profile.name
        
        val Matiz = Typeface.createFromAsset(assets, "fonts/Matiz.ttf")
        profileName.typeface = Matiz
        
        val PhosphateSolid = Typeface.createFromAsset(assets, "fonts/PhosphateSolid.ttf")
        btnPlayGame.typeface = PhosphateSolid
        btnGameRules.typeface = PhosphateSolid
        btnGameAuthors.typeface = PhosphateSolid
        btnSupportAuthors.typeface = PhosphateSolid
        
    }
    
    
    fun btnPlayGameClick(v: View) {
        val intent = Intent(this, StartGameActivity::class.java)
        intent.putExtra("profile", profile as Serializable)
        
        startActivity(intent)
    }
    
    fun btnGameRulesClick(v: View) {
        val intent = Intent(this, RulesActivity::class.java)
        startActivity(intent)
    }
}
